lib/pbench/cli/server/report.py

```diff
@@ -308,6 +308,78 @@ def report_cache(tree: CacheManager):
         )
 
 
+def columnize(items: dict[str, Any], width: int = 80):
+    line = ""
+    for item, count in sorted(items.items()):
+        if len(line) >= width:
+            click.echo(line)
+            line = ""
+        line += f"    {item:4d}: {count:>8,d}"
```

Comment:
Since the code adds to the line in chunks of 18 characters (I think...), the test at line 314 could produce a line which is "too long" (i.e., 90 characters, by default).  [I don't know why this doesn't happen in the same output in the description...did you specify a smaller width for that?]

I would recommend something like this:
```python
    line = ""
    for item, count in sorted(items.items()):
        addition = f"    {item:4d}: {count:>8,d}"
        if len(line) + len(addition) >= width:
            click.echo(line)
            line = ""
        line += addition
```
or
```python
    line = ""
    for item, count in sorted(items.items()):
        addition = f"    {item:4d}: {count:>8,d}"
        if len(line) + len(addition) >= width:
            click.echo(line)
            line = addition
        else:
            line += addition
```

Reply:
Yeah, I threw in the columnization at the last minute because the lists were rather long: you're right, it's not entirely accurate as is, but I didn't spend too many cycles worrying about it. However, I guess it's worth cleaning up.

(And, yeah, I think I generated the sample output with `--width 70` to make sure GitHub didn't wrap it. 😆 )